src/DIRAC/Interfaces/API/Job.py

```diff
@@ -83,7 +83,7 @@ def __init__(self, script=None, stdout='std.out', stderr='std.err'):
     self.stdout = stdout
     self.stderr = stderr
     self.logLevel = 'info'
-    self.executable = '$DIRACROOT/scripts/dirac-jobexec'  # to be clarified
+    self.executable = 'dirac-jobexec'  # to be clarified
```

Comment:
This change alone means that we'll need a new hackathon, I am afraid.

Reply:
I agree and I don't see a good way to avoid making this change.